src/utils/device.js

```diff
@@ -66,6 +66,11 @@ function isIOS () {
 }
 module.exports.isIOS = isIOS;
 
+function isMacOS () {
+  return /Mac/.test(window.navigator.platform);
```

Comment:
I would check `/Mac/i` just in case

Reply:
It seems to me that actually increases the risk of false detection, Apple / Safari is pretty consistent about correct case for Mac and iDevices

Reply:
Do you feel strongly about this one?  I was actually wondering if it would be better to look for `Mac OS` in `navigator.userAgent` than to check against values like `MacIntel` in `navigator.platform`